free-1.md

```diff
@@ -1,43 +1,53 @@
-# Free project 1
+# Music Mixing
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+This project is mostly targetted at people who have a general interest in audio mixing, but have neither the background nor the expensive software typically used (e.g. Ableton). Additionally, for the less serious music enthusiast, this DSL would be a significantly lighter weight and cheaper alternative to currently existing music software. This DSL would help users learn more about mixing music by providing them basic and easy to understand tools to manipualte sounds as well as allowing them to develop their own tools if necessary. For instance, some built-in features of the language might be basic volume adjustments at specified intervals or  allowing for a certain segment of a sound to be looped.
+
 
```

Comment:
Interesting, that sounds a lot like the audio playback program from our first lab. It would be interesting to see whether an entire project with the same idea would be structured in a similar or completely different manner!